src/main/java/org/springframework/data/elasticsearch/core/DefaultIndexOperations.java

```diff
@@ -60,22 +62,26 @@
  * 
  * @author Peter-Josef Meisch
  * @author Sascha Woo
+ * @author George Popides
  * @since 4.0
  */
 class DefaultIndexOperations extends AbstractDefaultIndexOperations implements IndexOperations {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(DefaultIndexOperations.class);
 
 	private final ElasticsearchRestTemplate restTemplate;
+	private final ResponseConverter responseConverter;
```

Comment:
this can be moved to the parent `AbstractDefaultIndexOperations` and be `protected`